src/pollution_model/run_medusa_2.py

```diff
@@ -661,28 +661,28 @@ def retrieve_input_parameters(scenario_id: int) -> Dict[str, Union[str, float]]:
     -------
     Dict[str, Union[str, float]]
         A dictionary with information selected from Rainfall MEDUSA 2.0 database based on scenario ID
-
-    Raises
-    -------
-    FileNotFoundError
-        Error raised if `medusa_scenarios` table is not found or does not contain the `scenario_id`.
     """
     # Connect to the database
     engine = setup_environment.get_database()
 
-    # Set up query command to pull information from medusa_scenarios table
-    query = text(
-        "SELECT * FROM medusa_scenarios WHERE scenario_id = :scenario_id"
-    ).bindparams(scenario_id=scenario_id)
-
     # Check table exists before querying
     if not check_table_exists(engine, 'medusa_scenarios'):
-        raise FileNotFoundError("medusa_scenarios table does not exist in database")
+        return None
 
-    # Get information by using scenario_id from medusa_scenarios table in the dataset
-    row = dict(engine.execute(query).fetchone())
+    else:
+        # Set up query command to pull information from medusa_scenarios table
+        query = text(
+            "SELECT * FROM medusa_scenarios WHERE scenario_id = :scenario_id"
+        ).bindparams(scenario_id=scenario_id)
+
+        # Get information by using scenario_id from medusa_scenarios table in the dataset
+        row = engine.execute(query).fetchone()
 
-    return row
+        # Return
+        if row is None:
+            return None
+        else:
+            return dict(row)
```

Comment:
Remove all the `else` here. It simplifies he level of indentation and makes the code easier to visually read.
The return is instant so the `else` will never be hit if it returns

Reply:
I have removed them.